[ENH] use schema to mention which "top directories" are allowed (#1289)
bids-standard/bids-specification: src/02-common-principles.md

A data type directory SHOULD NOT be defined if there are no files to be placed
in that directory.

### Other top level directories

> In addition of the subject subdirectories, the root directory of a BIDS dataset
> can also contain the following directories:

Reviewer: ```suggestion
In addition to the subject directories, the root directory of a BIDS dataset
MAY also contain the following directories:
```

A data type directory SHOULD NOT be defined if there are no files to be placed
in that directory.

### Other top level directories

In addition of the subject sub-directories, the root directory of a BIDS dataset
can also contain the following directories:

> 

Author: Definitely not sure about the wording here: suggestions more than welcome.

Reviewer 2: I'd use MAY instead of "can", other than that, it sounds good.

But it also made me notice that we sometimes use sub-directories and other times subdirectories in the spec. What should we use? my browser marks the word without hyphen as wrong, but Google shows me that both can be used. (very minor stuff here)

Reviewer 2: ```suggestion
### Other top level directories

In addition of the subject subdirectories, the root directory of a BIDS dataset
can also contain the following directories:

```
